lang/cs/Org.Apache.REEF.Client/Common/HttpClientHelper.cs

```diff
@@ -64,6 +69,19 @@ public string GetUrlResult(string url)
             return task.Result;
         }
 
+        ///// <summary>
+        ///// Get application Id
+        ///// </summary>
+        //public string AppId { get { return _appId; } }
+
+        /// <summary>
+        /// Get application status
+        /// </summary>
+        public State AppFinalStatus
```

Comment:
Yes, that is the entire change for. We need to allow the client to get Job status after the Job is submitted.